content/articles/wand-library-in-python/index.md

```diff
@@ -0,0 +1,151 @@
+
+### Introduction
+Wand library in Python contains a module known as [ImageMagick](https://imagemagick.org/index.php). It is an attribute developed to change the format of an image. ImageMagick is extensively used due to its capability to work with image formats in the broader scope and its precision and ease of use. The reason for studying this module in Python is to understand how you can manipulate a single image into various forms.
+
+Therefore, we will take a deep look at the wand Python binding, which Imagemagick developed in this article. Furthermore, with the help of several examples, which will well explain, we will learn about the features and applications of the Wand library in the Python programming language.
+### Prerequisite
+For a learner to have a complete understanding of this topic, they should have some knowledge of Python and should also have Python installed in their machine for easier running of code.
+### Table of contents
+- Defining Wand library
+- Installation Verification
+- How can we read an image in Python Wand?
+- Use Wand package in Python to blur an image
+- Image transform using Python wand
+- Use Python wand to draw an image
+- Conclusion
+### Defining Wand library
+Using Python programming language, which created a library known as Imagemagick, and this is what we term as the Wand. Using Wand library, images are manipulated in several ways which will be discussed later in the article. We can use the Wand to edit and open images, but in collaboration with NumPy, it offers significant functions for machine learning codes. Below are some of its uses.
+- We can use it to change the appearance of photographs.
+- This package additionally enhances the photos with various effects.
+- Image scaling and cropping are also supported.
+- The wand library can read and write images in various formats.
+
+We shall now take a look at how to install wand library.
+
+Let us check the following syntax, and in this case, we will use Ubuntu.
+```bash
+pip install Wand
+```
+The python module wand will be installed in the machine as a version of pip and Python.
+
+Again, we must include the ImageMagick dependencies because the wand library is an Imagick API. 
```

Comment:
Why do we need to add these dependencies? Wouldn't installing python package be enough?